### PR TITLE
Add blank=True to a bunch of fields for the admin

### LINKED ISSUE #594: Fix some of the admin pages having non-optional fields
A bunch of fields on the Puzzle model, for example, should be made optional

### PR DESCRIPTION
**Pull request by rgossiaux**
Fixes #594

### CODE REVIEW

**Comment by rawxfish** [comment]
shouldn't there be migration files too?

File: puzzles/models.py [hunk 1]
```diff
@@ -62,20 +62,23 @@ class Puzzle(models.Model):
...
+    tags = models.ManyToManyField(PuzzleTag, related_name="puzzles", blank=True)
 
-    metas = models.ManyToManyField("self", symmetrical=False, related_name="feeders")
+    metas = models.ManyToManyField(
+        "self", symmetrical=False, related_name="feeders", blank=True
+    )
 
     is_meta = models.BooleanField(default=False)
 
     active_users = models.ManyToManyField(
```
Comment (erwa) [inline puzzles/models.py:76]: not related to your change, but what was active_users for again?

**Review by erwa — APPROVED** [review]
LGTM

File: puzzles/models.py (hunk 1 above)
Comment (rgossiaux) [reply puzzles/models.py:76]: We're not actually using it, lol. I think the idea was to keep track of who's working on a puzzle at a time. We should probably go through and do some cleanup of the stuff we never ended up using or that we deprecated.

**Review by rawxfish — APPROVED** [review]

File: puzzles/models.py (hunk 1 above)
Comment (erwa) [reply puzzles/models.py:76]: lol I already forgot I added this feature back in #80...
yeah, doesn't seem like this is used anymore, we can probably just get rid of it

Comment (erwa) [reply puzzles/models.py:76]: Added #599 to clean up `active_users`.

**Pull request merged**